autopropose.service

```diff
@@ -4,7 +4,7 @@ Requires=docker.service
 After=docker.service
 
 [Service]
-ExecStart=/opt/rchain-testnet-node/autopropose
+ExecStart=/opt/rchain-testnet-node/autoproposeng.py
```

Comment:
Keep the filename please.

Reply:
> Keep the filename please.
OK.

> This is effectively 1:1 rewrite from Bash to Python.
I need it to evolve autopropose into what's been proposed at https://rchain.atlassian.net/wiki/spaces/OP/pages/752058391/Network+Hardening

> What complicated scenarios do you have in mind and how exactly does this help?
The "implementation" section at https://rchain.atlassian.net/wiki/spaces/OP/pages/752058391/Network+Hardening